synergy-design-system/synergy-design-system
# feat: ✨ progress-bar

# Pull Request

## 📖 Description

This PR adds the new `<syn-progress-bar />` element.

### 🎫 Issues

Closes #421

## 👩‍💻 Reviewer Notes

Just have a look at the provided Storybook documentation.

## 📑 Test Plan

- Check out this branch and build world.
- Spin up storybook
- Have a look at the new Components -> syn-progress-bar stories

## ✅ DoD
